### PR TITLE
added MorseCodeConverter and started test class

### PR DESCRIPTION
**Pull request by sarayaffe**

### CODE REVIEW

File: src/main/java/MorseCodeConverter.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    public String toMorseCode(String englishInput) {
+        StringBuilder morseString = new StringBuilder();
+        String[] array = englishInput.split(" ");
+
+        for (String s : array) {
+            for (int i = 0; i < s.length(); i++) {
+                try {
+                    morseString.append(code[s.charAt(i) - 'a']).append(" ");
+                } catch (Exception ignored) {}
```
Comment (andrewoid) [inline src/main/java/MorseCodeConverter.java]: Always (at least) call printStackTrace() when catching an Exception

File: src/main/java/MorseCodeConverter.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    public String toMorseCode(String englishInput) {
+        StringBuilder morseString = new StringBuilder();
+        String[] array = englishInput.split(" ");
+
+        for (String s : array) {
+            for (int i = 0; i < s.length(); i++) {
+                try {
+                    morseString.append(code[s.charAt(i) - 'a']).append(" ");
```
Comment (andrewoid) [inline src/main/java/MorseCodeConverter.java:33]: Your string shouldn't have a " " at the end.

File: src/main/java/MorseCodeConverter.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                } catch (Exception ignored) {}
+            }
+        }
+
+        return morseString.toString();
+    }
+
+    public String toEnglish(String morseCodeInput) {
```
Comment (andrewoid) [inline src/main/java/MorseCodeConverter.java:43]: What happens if you pass something uppercase?

File: src/main/java/MorseCodeConverter.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        StringBuilder englishString = new StringBuilder();
+        String[] array = morseCodeInput.split(" ");
+
+        for (String s : array) {
+            if(morseToEnglish.get(s) != null){
```
Comment (andrewoid) [inline src/main/java/MorseCodeConverter.java]: Don't call get(s) twice

File: src/test/java/MorseCodeConverterTest.java [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MorseCodeConverterTest {
+
+    @Test
+    public void toMorseCode(){
+        //given
+        MorseCodeConverter converter = new MorseCodeConverter();
```
Comment (andrewoid) [inline src/test/java/MorseCodeConverterTest.java]: Make this a member variable so all the test methods can use it.

File: src/main/java/MorseCodeConverter.java (hunk 2 above)
Comment (sarayaffe) [reply src/main/java/MorseCodeConverter.java:33]: how then do can you read the morse code

Comment (sarayaffe) [reply src/main/java/MorseCodeConverter.java:33]: *how do you read the morse code if its characters are not separated

Comment (andrewoid) [reply src/main/java/MorseCodeConverter.java:33]: I mean a space at the end of the WHOLE Morse code string

**Review by andrewoid — APPROVED** [review]

**Review by andrewoid — APPROVED** [review]

**Pull request merged**